quickwit/Cargo.lock

```diff
@@ -941,6 +941,14 @@ dependencies = [
  "serde",
 ]
 
+[[package]]
+name = "bitpacking"
+version = "0.8.3"
```

Comment:
:-/ how did we end up with two deps on bitpacking?

Reply:
Coming from this change https://github.com/quickwit-oss/tantivy/commit/0d4589219bcf8b4f10326e599c23ac102d28e42b

I think only temporary, until a new bitpacking release is done (I don't have access)